libs/agno/agno/tools/function.py

```diff
@@ -90,11 +91,9 @@ class Function(BaseModel):
     stop_after_tool_call: bool = False
     # Hook that runs before the function is executed.
     # If defined, can accept the FunctionCall instance as a parameter.
-    # Deprecated: Use tool_hooks instead.
```

Comment:
Did we actually un-deprecate this?

Reply:
Yes, this was incorrectly marked before.